aptos-move/e2e-tests/src/executor.rs

```diff
@@ -220,27 +214,22 @@ impl FakeExecutor {
         FakeExecutor::from_genesis(genesis.write_set())
     }
 
-    /// Create one instance of [`AccountData`] without saving it to data store.
-    pub fn create_raw_account(&mut self) -> Account {
-        Account::new_from_seed(&mut self.rng)
-    }
-
     /// Create one instance of [`AccountData`] without saving it to data store.
     pub fn create_raw_account_data(&mut self, balance: u64, seq_num: u64) -> AccountData {
-        AccountData::new_from_seed(&mut self.rng, balance, seq_num)
+        AccountData::new(balance, seq_num)
```

Comment:
we need deterministic key generation from seed, some of those accounts are written to golden files

Reply:
Ah makes sense, maybe I'll see if I'll introduce that in, otherwise, I'll leave it alone.